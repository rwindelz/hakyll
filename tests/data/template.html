<div>
    I'm so rich I have $$3.
    $echo test!$
    $body$
</div>
