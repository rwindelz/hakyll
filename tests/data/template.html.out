<div>
    I'm so rich I have $3.
    test!
    <p>This is an example.</p>
</div>
